PlanMyTrip/PlanMyTrip/View Controllers/DisplayResultsViewController.m

```diff
@@ -64,51 +71,68 @@ - (void)viewDidLoad {
     if(self.flightStatus == 1){
         self.FirstTitle.text = @"Flight Recomendations";
         self.firstThird = YES;
+        self.firstButton.backgroundColor = [UIColor whiteColor];
+        self.secondButton.backgroundColor = [UIColor whiteColor];
+        self.thirdButton.backgroundColor = [UIColor whiteColor];
+        self.firstFilter.backgroundColor = [UIColor whiteColor];
+        [self.firstFilter setTitle:@"Filter Results" forState:UIControlStateNormal];
         if(self.hotelStatus == 1){
             self.SecondTitle.text = @"Hotel Recomendations";
             self.secondThird = YES;
+            self.fourthButton.backgroundColor = [UIColor whiteColor];
+            self.fifthButton.backgroundColor = [UIColor whiteColor];
+            self.sixthButton.backgroundColor = [UIColor whiteColor];
+            self.secondFilter.backgroundColor = [UIColor whiteColor];
+            [self.secondFilter setTitle:@"Filter Results" forState:UIControlStateNormal];
             if(self.carStatus == 1){
                 self.ThirdTitle.text = @"Car Recomendations";
                 self.lastThird = YES;
+                self.seventhButton.backgroundColor = [UIColor whiteColor];
+                self.eighthButton.backgroundColor = [UIColor whiteColor];
+                self.ninthButton.backgroundColor = [UIColor whiteColor];
+                self.thirdFilter.backgroundColor = [UIColor whiteColor];
+                [self.thirdFilter setTitle:@"Filter Results" forState:UIControlStateNormal];
             }
         } else {
             if(self.carStatus == 1){
```

Comment:
also, self.carStatus itself is a BOOL, why not use it as BOOL? if (self.carStatus)